src/commands/debugging/initializeForDebugging.ts

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from 'vscode-azureextensionui';
+import { quickPickOS, quickPickPlatform } from '../../configureWorkspace/configUtils';
+import { DockerDebugScaffoldContext } from '../../debugging/DebugHelper';
+import dockerDebugScaffoldingProvider, { NetCoreScaffoldingOptions } from '../../debugging/DockerDebugScaffoldingProvider';
+import { DockerPlatform } from '../../debugging/DockerPlatformHelper';
+import { quickPickProjectFileItem } from '../../utils/quick-pick-file';
+import { quickPickWorkspaceFolder } from '../../utils/quickPickWorkspaceFolder';
+
+export async function initializeForDebugging(actionContext: IActionContext): Promise<void> {
+    const folder = await quickPickWorkspaceFolder('To configure Docker debugging you must first open a folder or workspace in VS Code.');
+    const platform = await quickPickPlatform();
+
+    let debugPlatform: DockerPlatform;
+    switch (platform) {
+        case '.NET Core Console':
+        case 'ASP.NET Core':
+            debugPlatform = 'netCore';
+            break;
+        case 'Node.js':
+            debugPlatform = 'node';
+            break;
+        default:
+    }
+
+    actionContext.telemetry.properties.platform = debugPlatform;
+
+    const context: DockerDebugScaffoldContext = {
+        folder: folder,
+        platform: debugPlatform,
+        actionContext: actionContext,
+    }
+
+    switch (context.platform) {
+        case 'netCore':
+            const options: NetCoreScaffoldingOptions = {
+                appProject: (await quickPickProjectFileItem(undefined, context.folder, 'You must choose a .NET Core project file to set up for Docker debugging.')).absoluteFilePath,

```

Comment:
>You [](start = 88, length = 3)

How about just "Choose a .NET Core project for Docker debugging"?

Reply:
It's not a prompt, rather it's the error message displayed if nothing is found. In fact I should probably change it to "No C# or F# project files were found."